omod-2.2/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_2/DrugResource2_2Test.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs2_2;
+
+import org.openmrs.Drug;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.webservices.rest.web.RestTestConstants1_11;
+import org.openmrs.module.webservices.rest.web.resource.impl.BaseDelegatingResourceTest;
+
+public class DrugResource2_2Test extends BaseDelegatingResourceTest<DrugResource2_2, Drug> {
```

Comment:
@mozzy11, did you tested your changes?

Reply:
noo , i havent yet written tests , 1 was still waiting for comments on the work so far

Reply:
Is it possible to construct unit tests for your changes as advised here https://wiki.openmrs.org/display/docs/Pull+Request+Tips ?

Reply:
its fine @Ruhanga , let me write some tests for the changes , just that i had first wanted to get some comments whether there are some properties i need to remove or to add to the resource .

Reply:
@Ruhanga , your long awaited tests are here  (: